pyproject.toml

```diff
@@ -134,7 +134,7 @@ filterwarnings = [
     "ignore:module 'sre_.*' is deprecated:DeprecationWarning",
     "ignore:Jitify is performing a one-time only warm-up",
 ]
-log_cli_level = "info"
+log_cli_level = "INFO"
```

Comment:
The SchemaStore schema for this expects upper case. You can see this with `uvx sp-repo-review[all] .` or at https://learn.scientific-python.org/development/guides/repo-review/.